packages/react-internal/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.styles.ts

```diff
@@ -344,6 +344,7 @@ export const getStyles = (props: IChoiceGroupOptionStyleProps): IChoiceGroupOpti
       },
       checked && {
         borderColor: iconOrImageChoiceBorderCheckedColor,
+        border: `1px solid ${focusBorderColor}`,
```

Comment:
I don't think this is the right place to set border. We want to have border when the option actually gained focus (not when `checked` is set to true)

I believe this is the line to start with, [here](https://github.com/microsoft/fluentui/blob/master/packages/react-internal/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.styles.ts#L290)

When you tab-focus on the selected option, you will see the focus border to show up. and that's the experience we want when user calls the `focus` public api

Reply:
my question would be, why didn't `focus` boolean on line 290 set to true when `focus()` is called?